src/handlers/diff_header.rs

```diff
@@ -479,49 +472,48 @@ mod tests {
     #[test]
     fn test_get_file_extension_from_marker_line() {
         assert_eq!(
-            get_file_extension_from_marker_line(
-                "--- src/one.rs	2019-11-20 06:47:56.000000000 +0100"
-            ),
-            Some("rs")
+            get_filename_from_marker_line("--- src/one.rs	2019-11-20 06:47:56.000000000 +0100"),
+            Some("one.rs")
         );
     }
 
     #[test]
     fn test_get_file_extension_from_diff_header_line() {
```

Comment:
Looks like this test name needs to be updated

Reply:
Done.